tests/system_tests_tcp_adaptor.py

```diff
@@ -1339,7 +1348,11 @@ def test_80_stats(self):
             assert "connectionsOpened" in output
             assert output["connectionsOpened"] > 0
             # egress_dispatcher connection opens and should never close
-            assert output["connectionsOpened"] == output["connectionsClosed"] + 1
+            if ncat_available():
+                conn_count_1 = output["connectionsOpened"] == output["connectionsClosed"] + 1
+                conn_count_2 = output["connectionsOpened"] == output["connectionsClosed"] + 2
+                # See the comments in test_20_tcp_connect_disconnect()
+                self.assertTrue(conn_count_1 or conn_count_2)
```

Comment:
```suggestion
                # See the comments in test_20_tcp_connect_disconnect()
                self.assertIn(output["connectionsOpened"], (output["connectionsClosed"] + 1, output["connectionsClosed"] + 2))
```

You _could_ do something like this, to avoid having to use assertTrue, but I'm not sure it is worthwhile improvement, all things considered...

Reply:
> You _could_ do something like this, to avoid having to use assertTrue, but I'm not sure it is worthwhile improvement, all things considered...

Done !